nextflow_schema.json

```diff
@@ -605,6 +631,27 @@
                     "fa_icon": "fas fa-align-center",
                     "enum": ["bwa", "bwamem2", "bwameme", "sentieon"]
                 },
+                "mt_aligner": {
+                    "type": "string",
+                    "default": "bwamem2",
+                    "description": "Specifies the alignment algorithm to use - available options are 'bwamem2' and 'sentieon'.",
+                    "fa_icon": "fas fa-align-center",
+                    "enum": ["bwa", "bwamem2", "sentieon"]
```

Comment:
available options are 'bwamem2' and 'sentieon'.

vs

"enum": ["bwa", "bwamem2", "sentieon"]